generator/internal/genclient/translator/openapi/openapi_test.go

```diff
@@ -72,14 +71,16 @@ func TestMakeAPI(t *testing.T) {
 		Documentation: "The response message for Locations.ListLocations.",
 		Name:          "ListLocationsResponse",
 		Fields: []*genclient.Field{
-			{Documentation: "A list of locations that matches the specified filter in the request.",
-				Name:     "locations",
-				Typez:    genclient.MESSAGE_TYPE,
-				Repeated: true},
-			{Documentation: "The standard List next-page token.",
-				Name:     "nextPageToken",
-				Typez:    genclient.STRING_TYPE,
-				Optional: true},
+			{
+				Name:          "locations",
+				Documentation: "A list of locations that matches the specified filter in the request.",
+				Typez:         genclient.MESSAGE_TYPE,
+				Repeated:      true},
+			{
+				Name:          "nextPageToken",
+				Documentation: "The standard List next-page token.",
+				Typez:         genclient.STRING_TYPE,
```

Comment:
ditto re: indenting between `true` and `}`

Reply:
Done